e2e2/test/cases/neuron/main_test.go

```diff
@@ -30,8 +31,21 @@ var (
 	neuronDevicePluginManifest []byte
 )
 
+func deployNvidiaDevicePlugin(ctx context.Context, config *envconf.Config) (context.Context, error) {
```

Comment:
```suggestion
func deployNeuronDevicePlugin(ctx context.Context, config *envconf.Config) (context.Context, error) {
```

and should this be with the other helpers?